playbooks/deploy.yaml

```
@@ -57,6 +60,11 @@
       - { type: host, database: all, user: all, address: '::1/128', auth_method: md5 }
   roles:
     - certificates
```

```suggestion
    - role: certificates
      when:
        - "certificate_source | default('default')  == 'default'"
```
or something like that (untested)?

---

Excellent ideas, added.